src/main/java/org/gridsuite/network/map/dto/definition/busbarsection/BusBarSectionFormInfos.java

```diff
@@ -23,4 +23,7 @@ public class BusBarSectionFormInfos extends ElementInfos {
     @JsonInclude(JsonInclude.Include.NON_NULL)
     private Integer horizPos;
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Why not create a BusBarSectionListInfos as for other equipment? So that we can use BusBarSectionListInfos in  toListInfos()

Reply:
Yes ok